### DISCUSSION TITLE
How to use spGFNn-xTB in DFTB+?

### DISCUSSION THREAD

**Discussion by RKS-labs** [tblite/tblite#144] [Q&A] [answered]
I found some spin constants in src/tblite/data/spin.f90. Is it enough to include them through `SpinConstants{}` in DFTB+ input files?

**Comment by haneug** [comment, marked as answer]
Yes this should be enough. You can also find the spin constants in the supporting information of our paper where they are better readable: https://doi.org/10.1002/jcc.27185
